### PR TITLE
style: unify eslintrc setup files

### PR DESCRIPTION
**Pull request by jesusprubio**
#### Please confirm that your PR fulfills these requirements
- [x] Multiple commits are squashed into one commit.
- [x] The commit message follows [Conventional Commits](https://github.com/IBM/kui/blob/master/CONTRIBUTING.md#conventional-commits), which allows us to autogenerate release notes; e.g. `fix(plugins/plugin-k8s): fixed annoying bugs`
- [x] All npm dependencies are pinned.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -38,5 +38,26 @@
...
+        "plugins/plugin-core-support/**/*.ts",
+        "plugins/plugin-editor/**/*.ts",
+        "plugins/plugin-grid/**/*.ts",
+        "plugins/plugin-manager/**/*.ts",
+        "plugins/plugin-openwhisk-debug/**/*.ts",
+        "plugins/plugin-openwhisk-editor-extensions/**/*.ts",
+        "plugins/plugin-proxy-support/**/*.ts",
+        "plugins/plugin-tutorials/**/*.ts",
+        "plugins/plugin-wrk/**/*.ts",
+        "plugins/plugin-wskflow/**/*.ts"
```
Comment (starpit) [inline .eslintrc.json]: good this be plugins/**/*.ts? don't we also want packages/**/*.ts?

Comment (jesusprubio) [reply .eslintrc.json]: The point is to add here one package/plugin, to fix the errors, make a PR and start with the next one.

If we add here all of them now we should have to fix all the errors in the same PR. Which would be the same that enabling the rule globally.

Comment (starpit) [reply .eslintrc.json]: i thought unused-vars was done? is the implication that it is not?

Comment (jesusprubio) [reply .eslintrc.json]: It isn't

Comment (abdonrd) [reply .eslintrc.json]: There are still many unused variables in the k8s, openwhisk & tekton plugins. And also in the packages.

**Pull request merged**